packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -341,13 +341,17 @@ const PRESETS = [
         name: 'Retail React App Test Project',
         description: '',
         templateSource: {
-            type: TEMPLATE_SOURCE_NPM,
-            id: '@salesforce/retail-react-app'
+            type: TEMPLATE_SOURCE_BUNDLE,
+            id: 'typescript-minimal'
         },
         questions: [...EXTENSIBILITY_QUESTIONS, ...RETAIL_REACT_APP_QUESTIONS],
         answers: {
-            ['project.extend']: true,
+            ['project.extend']: false,
             ['project.hybrid']: false,
+            ['project.extractAppExtensions']: true,
+            ['project.type']: 'PWAKitAppProject',
+            ['project.useApplicationExtensibility']: true,
+            ['project.selectedAppExtensions']: ['@salesforce/extension-chakra-storefront'],
```

Comment:
We aren't going to be using the old version of extensions via the "extends" config anymore.. so we are replacing this preset with answers that generate a project using all the extensions and also extracting the extensions into the project folder. This is probably the closest thing. 

Maybe we don't have to extract them?

But Adam has created a ticket to clean up the generator [here](https://gus.lightning.force.com/a07EE000026MzcVYAS)

Reply:
We also have the _store-locator_ extension. Do we want to add that now or later? For me, I'm ok with only the storefront for now, since it's the meatiest extension of them all.

Reply:
Yeah.. good catch I've added that and tested that the store locator works once I update the package.json with a valid config